### PR TITLE
ballistic armor module

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
what it literally says on the tin
40 bullet armor, nothing else

## Why It's Good For The Game
ff man bad, give module to counter ff man

## Changelog
:cl:
add: Added new ballistic armor module
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -189. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Sulaboy** [comment]
What about HvH, or events? Wouldn't this make you a bullet sponge?

**Comment by Pariah919** [comment]
> What about HvH, or events? Wouldn't this make you a bullet sponge?
>
the server isn't made around hvh and probably should never be
its just silly admin events

**Comment by Tennessee116** [comment]
> every normal gun should really get aim mode

> ff man bad, give module to counter ff man

Choose one.

**Comment by TGCMbetcher** [comment]
Make a mark2 for the experimental crate that gives 20 explosive armor as well

**Review by TiviPlus — APPROVED** [review]
Code ok

**Review by SplinterGP — APPROVED** [review]

**Comment by Sulaboy** [comment]
New EORD meta

**Comment by Pariah919** [comment]
no point to bothering

File: code/modules/modular_armor/attachables.dm [hunk 1]
```diff
@@ -185,3 +185,22 @@
...
 	return ..()
+
+/** Extra armor module */
+/obj/item/armor_module/attachable/ballistic_armor
+	name = "\improper Ballistic Armor Reinforcement"
+	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. A substantial amount of additional armor plating designed to fit inside some of the vulnerable portions of the Jaeger Combat Exoskeleton conventional armor patterns against bullets and nothing else. Will definitely impact mobility."
+	icon_state = "mod_ff_icon"
+	item_state = "mod_ff"
+	soft_armor = list("melee" = 0, "bullet" = 40, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	slowdown = 0.2
```
Comment (SplinterGP) [inline code/modules/modular_armor/attachables.dm:196]: i'd give it 0.1 slowdown myself, seeing its a module that makes your team not fucking bork you down, but this is fine and if it is not good we can buff/nerf it

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-243). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**